### ISSUE TITLE
Problem with registering yowsup requestcode

### ISSUE THREAD

**Issue by zykat** [tgalal/yowsup#1183]
Trying to register code I get the verification code successfully but after typing the received code I always get this error "An error occurred registering the registration code from WhatsApp. Reason: temporarily_unavailable", "retry_after" = 3600. 
After this period I always get "status: fail, reason: missing". 
I fix authentication bug in env_s40.py editing _VERSION = "2.13.21" but it doesn't solve registering problem.
Is there any issue or any update I need to do? 
##

**Comment by sbogeman** [comment]
I also have the exact same issue..even i try to add the parameter by edit the parameter in warrequest.py ( i thought missing the '-' chararter) but still failed.
Any one can help ?
I use yowsup v.24
yowsup-cli v2.0.13

**Comment by sbogeman** [comment]
hi zykat,

I got the same problem with you since yesterday , but this morning, i have success to retrieve the password.
Actually i also don't know what do i do . But this is what i do after it failed

1. Change the env to Android -- failed
2. Remove the installation of yowsup and re-install again
3. Request code sms -- success
4. Register again -- success.

**Comment by zykat** [comment]
I've tried to register code from two servers with different IPs and fresh yowsap installation. Still no success. And have now 
status: fail
retry_after: 81254
reason: too_recent" 
in addition...

**Comment by sbogeman** [comment]
Do you mean you got error 'too_recent' when you're asking new sms request code? or when register the code?
I also having this error, but this because of when i asked new request code after i have the new code again.
For your info: i use the default setting for the new installation, without change to Android env nor change the _Version. everything's default

**Comment by zykat** [comment]
I got error 'too_recent' asking new sms request code.

**Comment by LavanyaLea** [comment]
Hi sbogeman,
I had the same problem yesterday, trying to register the verification code...

[status] => fail
[reason] => temporarily _unavailable
[retry_after] => 3600

So I left it overnight, and this morning, it had a slightly different problem... so after:
yowsup-cli registration --requestcode sms --phone 44XXXXXXXXXX --cc 44 --mcc 234 --mnc 34

i got the code, and when i put:
yowsup-cli registration --register XXXXXX --phone 44XXXXXXXXXX --cc 44
it says 44 was unknown code!

When I tried to uninstall yowsup (sudo apt-get remove --auto-remove yowsup-cli), it said 
E: unable to locate package yowsup-cli

I'm running it on virtual machine, so in the end I uninstalled the whole ubuntu and reinstalled it, and after all that, I went back to square one: temporarily unavailable.

Any thoughts?
